sections/combat.tex

```diff
@@ -204,12 +204,10 @@ \subsection*{Campaign and Cooperative Combat}
 \end{multicols}
 
 \vspace*{\fill}
-\begin{scaledfigure}[blanker]
-  \centering
-  \includegraphics[width=\linewidth, height=\myspace, keepaspectratio]{\art/dread_knight.jpg}
-\end{scaledfigure}
 
-\clearpage
+\begin{center}
+  \includegraphics[width=0.97\linewidth]{\art/dread_knight.jpg}
```

Comment:
Not a big deal but to my surprise, I was able to squeeze Czech translation to a point where the Dread knight can be at full width, which looks slightly better. Maybe we could do the same with other languages

<img width="300" alt="Screenshot 2024-06-11 at 13 45 35" src="https://github.com/Heegu-sama/Homm3BG/assets/1928291/28025518-4545-49b9-8297-8b3c8422191d">